app/src/main/java/com/example/kotlin_mvp_calculator/mvp/presenter/CalculatorPresenter.kt

```diff
@@ -26,34 +53,73 @@ class CalculatorPresenter(val model: CalculatorModel, val view: CalculatorView)
                 when (clickEvent) {
                     RESET_EVENT -> {
                         model.reset()
-                        Log.d("PRESENTER", "reset_event!")
                     }
+
+                    DELETE_EVENT -> {
```

Comment:
are the brackets necessary? if not, remove them to clean the code pls

Reply:
done